mp/src/game/client/momentum/ui/MapSelection/BaseMapsPage.cpp

```diff
@@ -120,7 +121,7 @@ CBaseMapsPage::CBaseMapsPage(vgui::Panel *parent, const char *name) : PropertyPa
                                 ListPanel::COLUMN_UNHIDABLE | ListPanel::COLUMN_RESIZEWITHWINDOW);
     m_pMapList->AddColumnHeader(HEADER_MAP_LAYOUT, KEYNAME_MAP_LAYOUT, "#MOM_MapSelector_MapLayout",
 								GetScaledVal(30), GetScaledVal(30), GetScaledVal(30),
-								ListPanel::COLUMN_IMAGE | ListPanel::COLUMN_IMAGE_SIZETOFIT |
+                                ListPanel::COLUMN_IMAGE | ListPanel::COLUMN_IMAGE_SIZETOFIT |
```

Comment:
Not sure if this change was necessary, seems like the rest of the params list isn't shifted either.